Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -509,19 +509,25 @@
   - type: Faction
     factions:
     - Xeno
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      25: Critical
+      45: Dead
   - type: MeleeWeapon
+    range: 1.75
```

Comment:
NPCs will always attack from range 1 so probably don't want to consider range in balancing for now